_layouts/default.html

```diff
@@ -2,20 +2,20 @@
 <html>
     <head>
         <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-        <title>{{ page.title }}</title>
-        <meta name="viewport" content="width=device-width">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+        <title>{{ page.title }} | UWARC (VE3UOW)</title>
+        <!-- <meta name="viewport" content="width=device-width" /> -->
 
         <!-- syntax highlighting CSS -->
-        <link rel="stylesheet" href="{{ site.url }}/css/syntax.css">
+        <link rel="stylesheet" href="{{ site.url }}/css/syntax.css" />
```

Comment:
Not sure if you're aware, but the HTML5 spec actually frowns upon these closing slashes for void elements. Take a look at [this doc](http://dev.w3.org/html5/html-author/#void-elements).